src/IconToggle/IconToggle.react.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     /**
     * Name of icon to show
     */
-    name: PropTypes.string.isRequired,
+    name: PropTypes.string,
```

Comment:
`IconToggle` provides a ripple effect for `Icon`. However, we probably don't need both of them. But if we finally merge the MaterialIcons directly to this project (and remove react-native-vector-icons), it'll be very likely good to have this `Icon` component .. we'll see guys :)